src/components/note/__internal__/note.component.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Editor, EditorState } from 'draft-js';
+import tagComponent from '../../../utils/helpers/tags/tags';
+import {
+  StyledNote, NoteContent, InlineControl, Title, Footer, FooterContent
+} from './note.style.js';
+import StatusWithTooltip from './status-with-tooltip';
+
+const Note = (props) => {
+  const {
+    noteContent,
+    width,
+    inlineControl,
+    title,
+    name,
+    createdDate,
+    status,
+    ...rest
+  } = props;
```

Comment:
Why this
```js
 const {
    noteContent,
    width,
    ...rest
} = props
```
instead of this
```js
const Note = ({noteContent, width, ...rest})
```

Reply:
I was originally passing the props on to another component, that isn't needed anymore so I'll destructure in the top level as you suggest